app/helpers/spree/core/controller_helpers/flow_io_order_helper_decorator.rb

```diff
@@ -33,6 +33,7 @@ def adjust_zone_and_ip
               session['_f60_session'] = order_flow_session_id
               session['_f60_expires_at'] = order_session_expired
             end
+            cookies['_f60_session'] = { value: order_flow_session_id, domain: '.mejuri.com' }
```

Comment:
I think you should move this line into the `elsif` after line 34

And still add a similar line into `flow_zone` method of current_zone_loader_decorator.

And then, also,  in the OrderSync.

Reply:
Sorry Aurel I never answered you here. 

On relation to moving that inside de `elsif` sure! lets do the change! I placed outside the if/elsif to make sure to always have that cookie set. but it is true that placing that there is overkill.

On relation to adding a similar line into the `flow_zone` method, I initially thought about that as well, but since this `adjust_zone_and_ip` method is called always I thought that probably was not necessary to add it in the `flow_zone` method to avoid duplicating the call; However, I am not completely opposed to adding this call in that method. 

I will push these changes for now so we can test them properly :) Thanks!